src/defines/magick-define.ts

```diff
@@ -0,0 +1,14 @@
+import {IDefine} from './define';
+import {MagickFormat} from '../magick-format';
+
+export class MagickDefine implements IDefine {
+  format: MagickFormat;
```

Comment:
Could you please add `private ` here to be in line with the rest of the code?

Reply:
Do you mean like this:
```
private format: MagickFormat;
```
`format` is public in `IDefine` interface. I saw instead that in `DefinesCreator` I make `format` as public but in Magick.NET it's `protected`. Should I make it private or protected?

Reply:
Of course this should be public for the interface thanks for pointing that out. Maybe we should mark the fields as `readonly` in the interface and implementation instead?

Reply:
Yes, I marked those fields as readonly